net-p2p/gridcoin/gridcoin-5.3.3.0.ebuild

```
@@ -0,0 +1,164 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic systemd desktop
+
+DESCRIPTION="Gridcoin Proof-of-Stake based crypto-currency that rewards BOINC computation"
+HOMEPAGE="https://gridcoin.us/"
+
+if [[ ${PV} == 9999 ]]; then
+		inherit git-r3
+		EGIT_REPO_URI="https://github.com/gridcoin/Gridcoin-Research.git"
+		EGIT_BRANCH="development"
+		PATCHES="${FILESDIR}/gridcoin-9999-desktop.patch"
+else
+		SRC_URI="https://github.com/gridcoin/Gridcoin-Research/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+		KEYWORDS="~amd64"
+		S="${WORKDIR}/Gridcoin-Research-${PV}"
+fi
+
+SLOT="0"
+LICENSE="MIT"
+
+IUSE_GUI="qt5 dbus"
+IUSE_DAEMON="daemon"
+IUSE_OPTIONAL="+boinc qrcode upnp pic libraries utils +harden bench ccache static debug test"
+IUSE="${IUSE_GUI} ${IUSE_DAEMON} ${IUSE_OPTIONAL}"
+
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="|| ( daemon qt5 ) dbus? ( qt5 ) qrcode? ( qt5 )"
+
+RDEPEND="
+	acct-group/gridcoin
+	acct-user/gridcoin
+	>=dev-libs/boost-1.73.0:=
+	dev-libs/libevent
+	>=dev-libs/libzip-1.3.0:=
+	>=dev-libs/openssl-1.1.1d:=
+	net-misc/curl
+	sys-libs/db:5.3[cxx]
+	boinc? ( sci-misc/boinc )
+	dbus? ( dev-qt/qtdbus:5 )
+	qrcode? ( media-gfx/qrencode )
+	qt5? ( dev-qt/qtcore:5 dev-qt/qtgui:5 dev-qt/qtnetwork:5 dev-qt/qtwidgets:5 dev-qt/qtconcurrent:5 dev-qt/qtcharts:5 )
```

```suggestion
	qt5? (
		dev-qt/qtcore:5
		dev-qt/qtgui:5
		dev-qt/qtnetwork:5
		dev-qt/qtwidgets:5
		dev-qt/qtconcurrent:5
		dev-qt/qtcharts:5
	)
```

But it looks like this should be [USE=gui instead](https://projects.gentoo.org/qa/policy-guide/use-flags.html#pg0802)?